client/components/Editor/ConfirmableDeleteButton.jsx

```diff
@@ -1,40 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Confirm } from 'semantic-ui-react';
+import { Confirm, Menu, Icon } from 'semantic-ui-react';
 
 class ConfirmableDeleteButton extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             open: false
         };
+
+        this.onClick = this.onClick.bind(this);
+        this.onCancel = this.onCancel.bind(this);
+        this.onComplete = this.onCancel;
```

Comment:
Why does `onComplete` have to be named differently, if it just references `onCancel`